source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/ReportDashboard.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@model CommonJobs.Mvc.UI.Areas.Evaluations.Models.ReportDashboard
+@{
+    ViewBag.Title = "Dashboard Recursos Humanos";
+    Layout = "~/Areas/Evaluations/Views/Shared/_Layout.cshtml";
+    ScriptManager.RegisterCss(Url.Content("~/Content/twitter-bootstrap/less/bootstrap.less"), priority: 900);
+    ScriptManager.RegisterCss(Url.Content("~/Content/twitter-bootstrap/less/responsive.less"), priority: 900);
+    ScriptManager.RegisterCss(Url.Content("~/Content/Evaluations/evaluations.css"));
+    ScriptManager.RegisterGlobalJavascript("baseUrl", Url.Content("~"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/jquery-1.7.2.min.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/underscore.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Content/twitter-bootstrap/js/bootstrap-typeahead.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Content/twitter-bootstrap/js/bootstrap-modal.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/url-generator.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/knockout-2.2.0.js"), priority: 900);
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/Evaluations/common.js"));
+    ScriptManager.RegisterJs(Url.Content("~/Scripts/Evaluations/dashboard-human-resources.js"));
+}
+
+<div class="content-evaluation dashboard recursos-humanos">
+    <h2 class="title">Historial de Evaluaciones</h2>
+    <table class="table">
+        <thead>
+            <tr data-bind="foreach: headers">
+                <!-- ko if: sortable -->
+                <th data-bind="attr: { class: 'col-' + ($index()+1) }, click: $parent.sort, css: { selected: activeSort}">
```

Comment:
Sounds good, I believe that will impact on all template, because they is used on all templates or on almost all... Maybe will be a good improvement!